nilearn/datasets/__init__.py

```diff
@@ -19,4 +19,5 @@
            'fetch_harvard_oxford', 'fetch_atlas_msdl', 'fetch_msdl_atlas',
            'fetch_atlas_power_2011', 'fetch_atlas_smith_2009',
            'fetch_smith_2009', 'fetch_atlas_yeo_2011',
-           'fetch_yeo_2011_atlas', 'fetch_mixed_gambles']
+           'fetch_yeo_2011_atlas', 'fetch_mixed_gambles',
+           'fetch_atlas_aal_spm_12']
```

Comment:
About exposing fetch_atlas_aal_spm12, you want to add fetch_atlas_aal_smp12 in the import from .atlas a few lines above so that you can do:

```python

from nilearn import datasets
datasets.fetch_atlas_aal_smp12()
```

For the record `__all___` is used when doing `from nilearn.datasets import *`.